src/start/dropping-packets.md

```diff
@@ -69,7 +69,7 @@ sudo ./target/debug/myapp --path ./target/bpfel-unknown-none/debug/myapp --iface
 ### Testing the Allowed IP addresses
 
 ```console
-curl --interface eth0 192.168.0.22
+curl eth0 192.168.0.22
```

Comment:
This won't work, it's not a valid curl invocation. Why is passing the interface needed at all?

Reply:
Ah shoot. My bad I meant to delete that part as well. Will amend and push